chat-plugins/info.js

```diff
@@ -1833,145 +1833,84 @@ var commands = exports.commands = {
+		if (showSeasonal) return this.parse('/formathelp seasonal');
+		return this.parse('/formathelp ' + target);
 	},
 	othermetashelp: ["/om - Provides links to information on the Other Metagames.",
 		"!om - Show everyone that information. Requires: + % @ # & ~"],
 
-	/*formats: 'formathelp',
+	banlists: 'formathelp',
+	tier: 'formathelp',
+	tiers: 'formathelp',
+	formats: 'formathelp',
+	tiershelp: 'formathelp',
 	formatshelp: 'formathelp',
 	formathelp: function (target, room, user) {
 		if (!this.canBroadcast()) return;
-		if (this.broadcasting && (room.id === 'lobby' || room.battle)) return this.sendReply("This command is too spammy to broadcast in lobby/battles");
+		if (!target) {
+			return this.sendReplyBox(
+				"- <a href=\"https://www.smogon.com/tiers/\">Smogon Tiers</a><br />" +
+				"- <a href=\"https://www.smogon.com/forums/threads/3498332/\">Tiering FAQ</a><br />" +
+				"- <a href=\"https://www.smogon.com/xyhub/tiers\">The banlists for each tier</a><br />" +
+				"<br /><em>Type /formatshelp <strong>[format|section]</strong> to get details about an available format or group of formats.</em>"
+			);
+		}
+		var targetId = toId(target);
+		if (targetId === 'ladder') targetId = 'search';
+		if (targetId === 'all') targetId = '';
+
+		var formatList;
+		var format = Tools.getFormat(format);
+		if (format.effectType === 'Format') formatList = [format.id];
+		if (!formatList) {
+			if (this.broadcasting && (room.id === 'lobby' || room.battle)) return this.sendReply("This command is too spammy to broadcast in lobby/battles");
+			formatList = Object.keys(Tools.data.Formats).filter(function (formatid) {return Tools.data.Formats[formatid].effectType === 'Format';});
+		}
+
+		// Filter formats and group by section
+		var sections = {};
+		for (var i = 0; i < formatList.length; i++) {
+			var format = Tools.getFormat(formatList[i]);
+			var sectionId = toId(format.section);
+			if (targetId && !format[targetId + 'Show'] && sectionId !== targetId && !format.id.startsWith(targetId)) continue;
+			if (!sections[sectionId]) sections[sectionId] = {name: format.section, formats: []};
+			sections[sectionId].formats.push(format.id);
+		}
+
+		// Build tables
 		var buf = [];
-		var showAll = (target === 'all');
-		for (var id in Tools.data.Formats) {
-			var format = Tools.data.Formats[id];
-			if (!format) continue;
-			if (format.effectType !== 'Format') continue;
-			if (!format.challengeShow) continue;
-			if (!showAll && !format.searchShow) continue;
-			buf.push({
-				name: format.name,
-				gameType: format.gameType || 'singles',
-				mod: format.mod,
-				searchShow: format.searchShow,
-				desc: format.desc || 'No description.'
-			});
+		for (var sectionId in sections) {
+			buf.push("<h3>" + Tools.escapeHTML(sections[sectionId].name) + "</h3>");
+			buf.push("<table border=\"2\" cellspacing =\"0\" cellpadding=\"5\"><thead><th>Name</th><th>Mode</th><th>Description</th></thead><tbody>");
```

Comment:
1px gray border is a good start, that'll make it look like it's from 10 years ago instead of 20 years ago.